content/docs/command-reference/repro.md

```diff
@@ -1,32 +1,37 @@
 # repro
 
-Reproduce complete or partial [pipelines](/doc/command-reference/dag) by
-executing commands defined in their [stages](/doc/command-reference/run) in the
-correct order. The commands to be executed are determined by recursively
-analyzing dependencies and <abbr>outputs</abbr> of the target stages.
+Reproduce complete or partial <abbr>pipelines</abbr> by executing commands
+defined in their [stages](/doc/command-reference/run) in the correct order. The
+commands to be executed are determined by recursively analyzing dependencies and
+<abbr>outputs</abbr> of the target stages.
 
 ## Synopsis
 
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
-                 [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull] [--glob]
+                 [targets [<target> ...]]
 
 positional arguments:
-  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
-                 directories to search for stages can also be given.
+  targets       Limit command scope to these .dvc or dvc.yaml files,
+                or stage names.
 ```
 
+> See [`targets`](#targets-argument-optional) for more details.
+
 ## Description
 
-`dvc repro` provides a way to regenerate data pipeline results, by restoring the
-dependency graph (a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph))
-implicitly defined by the stages listed in `dvc.yaml`. The commands defined in
-these stages can then be executed in the correct order, reproducing pipeline
-results.
+Provides a way to regenerate data pipeline results, by restoring the dependency
+graph (a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) implicitly
+defined by the stages listed in `dvc.yaml`. The commands defined in these stages
+are then be executed in the correct order.
+
+For stages with multiple commands (having a list in the `cmd` field), commands
+are run one after the other in the order they are defined. The failure of any
+command will halt the remaining stage execution, and raises an error.
 
-> Pipeline stages are defined in a `dvc.yaml` file (either manually or by using
```

Comment:
does not look relevant, please let's scope this PR to the `--glob` thingy

Reply:
Hmmm. This is (losely) related to the `targets` argument, which is what this PR is actually on at least since I took it over.

Reply:
But OK, rolled back.